docker/multi-process/scripts/init

```diff
@@ -65,6 +72,16 @@ case "${DATABASE_ADAPTER}" in
   *) echo "Unsupported database adapter. Available adapters are mysql2, and postgresql." && exit 1 ;;
 esac
 
+# Fixup the Procfile and prepare the PORT
+if [ -n "${DO_NOT_RUN_JOBS}" ]; then
+  perl -pi -e 's/^jobs:/#jobs:/' /app/Procfile
+fi
+
+IP="${IP:-0.0.0.0}"
+PORT="${PORT:-3000}"
+PORT="$(echo $PORT|awk -F ':' '{print $NF}')"
```

Comment:
For which use case is that replacement required?

Reply:
In Kubernetes, if the service is called huginn, an environment variable is set up "HUGINN_PORT =tcp://xxx.xxx.xxx.xxx:3000", which gets processed by the `init` script to `.env` as `PORT`.

Reply:
I saw this format when working on GitLab CI images and was wondering why they would put a whole connection string in a variable named `_PORT`. Is there an other way to get the port? That solution looks very specific to Kubernetes and stops working if the service is called `my_huginn`, right?

Reply:
It was actually trying to handle an unintended case. Everything defaults to use port 3000, so even if `PORT` or `HUGINN_PORT` is not specified, things will work fine (provided a mechanism such as `-p 3000:3000` is set up to forward it). It was natural to call the service "huginn", but that environment variable messes the setup.